Jenkinsfile

```diff
@@ -144,13 +179,24 @@ pipeline {
 					# kill lisk-service process
 					make -C lisk-service logs
 					make -C lisk-service down
+					
+					# kill enevti-service process
+					make -C enevti-service logs
+					make -C enevti-service down
 
 					# kill lisk-core process
 					kill $( cat lisk-core.pid ) || true
 					sleep 10
 					kill -9 $( cat lisk-core.pid ) || true
 					cat lisk-core.out
 					cat lisk-core.err
+					
+					## kill enevti-core process
+					# kill $( cat enevti-core.pid ) || true
+					# sleep 10
+					# kill -9 $( cat enevti-core.pid ) || true
+					# cat enevti-core.out
+					# cat enevti-core.err
```

Comment:
Since enevti-service and lisk-core were shut down properly, why wasn't enevti-core shut down?

Reply:
Reason was because for now the enevti-core isn't running at the moment(there is a sight issue with enevti running on jeinkins which I would be resolving in this [issue](https://github.com/LiskHQ/lisk-desktop/issues/5393#issue-1945301032)) because of those reasons, there would be no pid, out or err files created. Reason for me commenting those lines out @ikem-legend